inst/include/sampler.h

```diff
@@ -154,12 +154,12 @@ namespace smc {
         ///Returns the current particle set stored in the history.
         population<Space> GetHistoryPopulation(long n) const {return History[n].GetValues();}
         ///Returns a reference to the particle set stored in the history.
-        population<Space> & GetHistoryPopulationRefs(long n) {return History[n].GetValues();}
+        population<Space> & GetHistoryPopulationRefs(long n) const {return History[n].GetValues();}
```

Comment:
I'd be inclined to return a `const population<Space> &` here, to avoid unintended consequences: I don't see this as a way for facilitating changing the history of the sampler from externally.